### ISSUE TITLE
Double-digit argument lists are processed incorrectly

### ISSUE THREAD

**Issue by bobbygryzynger** [reconquest/shdoc#48]
When I create a documentation block like this:

```sh
# @description This function has a lot of arguments.
#
# @arg $1 string Arg 1
# @arg $2 string Arg 2
# @arg $3 string Arg 3
# @arg $4 string Arg 4
# @arg $5 string Arg 5
# @arg $6 string Arg 6
# @arg $7 string Arg 7
# @arg $8 string Arg 8
# @arg $9 string Arg 9
# @arg $10 string Arg 10
my_function() {...}
```

The double-digit argument (`$10`) is not processed correctly. The output I get is: 
```md
### my_function()

This function has a lot of arguments.

#### Arguments

* **$1** (string): Arg 1
* **$2** (string): Arg 2
* **$3** (string): Arg 3
* **$4** (string): Arg 4
* **$5** (string): Arg 5
* **$6** (string): Arg 6
* **$7** (string): Arg 7
* **$8** (string): Arg 8
* **$9** (string): Arg 9
* $10 string Arg 10
```

**Comment by bobbygryzynger** [comment]
#41 also mentions this.

**Comment by bobbygryzynger** [comment]
Fixed in #63.
